play/src/i18n/en-US/chat.ts

```diff
@@ -42,10 +42,11 @@ const chat: BaseTranslation = {
     searchUser: "Search for user, map, etc ...",
     searchChat: "Search for channel, message, etc ...",
     people: "People",
-    rooms: "Rooms",
+    rooms: "Your Discussions",
     addNew: "Add new : ",
     invitations: "Invitations",
     suggestedRooms: "Suggested rooms",
+    joinableRooms: "Salons rejoignables",
     availableRooms: "Available rooms",
```

Comment:
What is the difference between "availableRooms" and "joinableRooms"? It is really clear to the user?